quarkus/server/src/main/java/org/keycloak/quarkus/_private/IDELauncher.java

```diff
@@ -33,6 +35,6 @@ public static void main(String[] args) {
             devArgs.add("start-dev");
         }
 
-        Quarkus.run(devArgs.toArray(new String[devArgs.size()]));
+        KeycloakMain.main(devArgs.toArray(new String[0]));
```

Comment:
may I ask why? :)

Reply:
Easier to dev/test CLI. I can revert this change as it is not really needed.